components/MainButton/MainButton.styles.js

```diff
@@ -15,7 +15,8 @@ const styles = StyleSheet.create({
 		width: '100%',
 		paddingTop: '5%',
 		paddingBottom: '5%',
-		borderRadius: 10,
+		borderRadius: 20,
+		marginBottom: '-10%'
```

Comment:
Be careful editing components, it can affect the whole application. But it's fine, I can fix this on my pull request since I added more props for the button styling.